modules/ROOT/pages/how-a-bdm-is-deployed.adoc

```diff
@@ -205,44 +205,3 @@ classloader (and in the process classloaders of all processes on this tenant), a
 Then the new BDM is directly available, even if in most cases, new (versions of) processes must be (re-)deployed to take
 advantage of the BDM.
 
-
```

Comment:
About what is above "Deploying a BDM via the Bonita Portal": this is a duplicate from what is already explained in the page "bdm-management-in-bonita-bpm-portal". This is not technical.
We should compare both content and merge that all in the above cited page.

Reply:
Only rows 200 and 201 are technical and should be kept for this page, and should not be mentioned in the BDM management page

Reply:
Absolutely not:
* bdm-management-in-bonita-bpm-portal explains functionally how to use the BDM through the portal
* how-a-bdm-is-deployed is a technical page that explains how it works behind the hood

Reply:
We totally agree: what I wrote and what you wrote mean the same: functional explanation should go to bdm-management-in-bonita-bpm-portal; technical content shoud stay in how-a-bdm-is-deployed.

Reply:
So this is ok like that, right?